strawberry/schema/types/base_scalars.py

```diff
@@ -23,7 +23,7 @@ def inner(value: str):
 
 def parse_decimal(value: str) -> decimal.Decimal:
     try:
-        return decimal.Decimal(value)
+        return decimal.Decimal(str(value))
```

Comment:
According to the type annotations on the function signature, `value` should already be a `str`. Is that erroneous?

Reply:
Oh, you just commented lol

Reply:
yes, sorry I left a comment above ^ the typehint is wrong. I would change it to something else